dapps/evancore.vue.libs/src/vue-core.ts

```diff
@@ -83,15 +83,15 @@ export async function initializeVue(options: EvanVueOptionsInterface) {
       dapp: dappToLoad,
       uiState: {
         profile: {
-          selectedSharedContacts: [],
+          selectedSharedContacts: null,
```

Comment:
why changing the initial type? This would make it very hard to handle in components since everywhere needs to be checked whether it's `null` where we use methods like .length, .filter, .indexOf ... Do we need a semantically difference for null selected and none selected?

Reply:
we store currently an account id only not an array of ids